src/libraries/System.CodeDom/tests/System/CodeDom/Compiler/CompilerResultsTests.cs

```diff
@@ -31,7 +31,7 @@ public void Ctor_TempFileCollection(TempFileCollection tempFiles)
         [Fact]
         public void CompiledAssembly_GetWithPathToAssemblySet_ReturnsExpectedAssembly()
         {
-            var results = new CompilerResults(null) { PathToAssembly = typeof(CompilerResultsTests).Assembly.Location };
+            var results = new CompilerResults(null) { PathToAssembly = (PlatformDetection.IsBrowser ? "/" : string.Empty) + typeof(CompilerResultsTests).Assembly.Location };
```

Comment:
Is this a bug? Where the path is not rooted?

Reply:
There is the following failure:
```
        <failure exception-type="System.ArgumentException">
          <message><![CDATA[System.ArgumentException : Path \"System.CodeDom.Tests.dll\" is not an absolute path. (Parameter 'path')]]></message>
          <stack-trace><![CDATA[   at System.Reflection.Assembly.LoadFile(String path)
   at System.CodeDom.Compiler.CompilerResults.get_CompiledAssembly()
   at System.CodeDom.Compiler.Tests.CompilerResultsTests.CompiledAssembly_GetWithPathToAssemblySet_ReturnsExpectedAssembly()
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)]]></stack-trace>
        </failure>
```

Reply:
RIght, so shouldn't `Assembly.Location` return an absolute path?

Reply:
Not currently on wasm.  We're still investigating.

Reply:
Yes, it should.

Reply:
I believe that someone is addressing this issue. Btw we already have the similar workaround, e.g. here https://github.com/dotnet/runtime/pull/39306/files#diff-b8c7220f6cfb2e8341450e98ff7cd5f2R21

Reply:
I see. I just wanted to make sure if we had an issue tracking this so that we don't accidentally miss it and just workaround it on the tests.